DSL/Search.php

```diff
@@ -69,7 +69,7 @@ class Search
     /**
      * @var string|null
      */
-    private $scrollDuration;
+    private $scroll;
```

Comment:
First of all $scrollDuration is a private field. It has a getter getScroll() and a setter setScroll(). So is it really a bad thing to rename it to simply $scroll ? I've changed its name in order to be able to test its getters and setters using EncapsulationTestAwareTrait.